docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/pod-security-standards.md

```diff
@@ -2,67 +2,73 @@
 title: Pod Security Standards
 ---
 
-[Pod Security Standards (PSS)](https://kubernetes.io/docs/concepts/security/pod-security-standards/) define security restrictions for a broad set of workloads.  They replace [Pod Security Policies](https://kubernetes.io/docs/concepts/security/pod-security-policy/), which are now deprecated.
-
+[Pod Security Standards (PSS)](https://kubernetes.io/docs/concepts/security/pod-security-standards/) define security restrictions for a broad set of workloads. They replace [Pod Security Policies](https://kubernetes.io/docs/concepts/security/pod-security-policy/) in Kubernetes v1.25 and above.
```

Comment:
```suggestion
[Pod Security Standards (PSS)](https://kubernetes.io/docs/concepts/security/pod-security-standards/) and [Pod Security admissions (PSA)](https://kubernetes.io/docs/concepts/security/pod-security-admission/) define security restrictions for a broad set of workloads. They replace [Pod Security Policies](https://kubernetes.io/docs/concepts/security/pod-security-policy/) in Kubernetes v1.25 and above.

PSS define security levels for workloads.  PSAs describe requirements for pod security contexts and related fields. PSAs reference PSS levels to define security restrictions.
```

I had a look at https://github.com/rancher/rancher-docs/issues/462, started writing, and realized that the issue overlaps with content in this PR. On this page, we move quickly from PSS to discussing PSA config templates, and on reflection, we need a little more of an explanation of how PSS and PSA relate, so that the section on templates feels more connected.

Reply:
In addition -- this screenshot of the GUI: 
![image](https://user-images.githubusercontent.com/19174201/224775502-1c5d0a03-061c-4fe1-9309-eef7d57c6ea0.png)

The warning in the screenshot states that PSA is the replacement for PSPs, and doesn't mention PSS. Maybe this page should be retitled to Pod Security Admissions & Pod Security Standards, so that it's in sync with the GUI?